### DISCUSSION TITLE
Dragging dynamically loaded image to dropzone on the same page does nothing, dataTransfer files array is empty

### DISCUSSION THREAD

**Discussion by chrxmvtik** [react-dropzone/react-dropzone#1313] [Q&A]
Salut fellas,

Recently I go into react-dropzone and I wanted to test it out, however, in my scenario it doesn't really work everywhere.

This is my image component, its source is being loaded when I type url in the form - it loads fine.
```js
<img
  id="image-preview"
  src={imageWatcher}
  width={230}
  height={230}
  className="p-3 px-5 py-2.5 mr-2 mb-2"
  alt={imageWatcher}
  draggable
  onDragStart={(e) => handleDragStartEvent(e, 'image-preview')}
/>
```
Then right next to it I have dropzone component
```js
<Dropzone onDrop={acceptedFiles => console.log(acceptedFiles)}>
  {({getRootProps, getInputProps}) => (
    <section>
      <div {...getRootProps()}>
        <input {...getInputProps()} />
        <p>Drag 'n' drop some files here, or click to select files</p>
      </div>
    </section>
  )}
</Dropzone>
```

When I drag image from my filesystem to browser onto Dropzone component - it works great.

However when I try to grab dynamically loaded image ('image-preview'), then no files are being transfered to dropzone component - it works only using Safari, when using Edge or Chrome, it doesn't work.

Any ideas?

**Comment by johncalvinroberts** [comment]
I've also ran into this same issue. Did you find a solution eventually @chrxmvtik ?

**Comment by rolandjitsu** [comment]
@chrxmvtik and @johncalvinroberts this package is only meant for use with dragging and dropping (or selecting with the file picker) from your file system, and not from the browser window.

Maybe https://react-dnd.github.io/react-dnd/about is a more appropriate choice?

Alternatively, you could look into adding support for such a feature to this package and I'd be more than happy to review the PR.
